docs/upgrade-notes/upgrade_rds_phase_3_release.md

```diff
@@ -0,0 +1,156 @@
+---
+id: upgrade-rds-phase-3-release
+title: Upgrade RDS Phase 3 Release
+hide_title: false
+---
+
+## Background
+
+Release v16 of Cumulus Core includes an update to remove the now-unneeded AWS DynamoDB tables for the primary archive, as this datastore has been fully migrated to PostgreSQL databases in prior releases, and should have been operating in a parallel write mode to allow for repair/remediation of prior issues.
+
+## Requirements
+
+To update to this release (and beyond) users must:
+
+- Have deployed a release of at least version 11.0.0 (preferably at least the latest supported minor version in the 11.1.x release series), having successfully completed the transition to using PostgreSQL as the primary datastore in release 11
```

Comment:
The more I think about it the more it seems like it would be far preferable if users went to v15 before taking this update. If they're starting on v11 they'll need to take a postgres update, a Node update, _several_ more minor updates, AND this. That seems like a lot of risk. It's up to integrators in the end though. Maybe we just strongly suggest they update to v15 (or something in the ballpark) first?

Reply:
`The more I think about it the more it seems like it would be far preferable if users went to v15 before taking this update.`

We can make that assertion, but for the moment, we are required to support v11.1.x->P3 as 11.1.x is the current supported version.

Reply:
Yes, absolutely. But what if, as a part of our instructions here, we suggest that they update first to v15 (for example). Maybe that just makes it too complicated though. They'll need to deal with the breaking changes either way.

Reply:
It probably doesn't matter. We don't need to prescribe how they go about the upgrade. If they want to do it all in one go from v11 > 15, that's fine. We support it.